File: 연습장/202308/10_목/14. 핸드폰 번호 가리기.py
"""
* Title: 핸드폰 번호 가리기
https://school.programmers.co.kr/learn/courses/30/lessons/12948
"""
def solution(phone_number):
    length = len(phone_number)
    back = phone_number[-4:]
    answer = '*' * (length - 4) + back

    return answer

File: _프로그래머스 js/14. 핸드폰 번호 가리기.js
/*
* Title: 핸드폰 번호 가리기
https://school.programmers.co.kr/learn/courses/30/lessons/12948
*/
const solution = (phone_number) => {
  const len = phone_number.length;
  const head = Array(len - 4).fill('*');
  const tailor = phone_number.substring(len - 4);

  return `${head.join('')}${tailor}`;
};

// "*******4444"
console.log(solution('01033334444'));
